build/_build/PrepareRelease/GenerateIntegrationDefinitions.cs

```diff
@@ -73,18 +66,77 @@ public static void Run(ICollection<string> assemblyPaths, params string[] output
                                                            .ToArray(),
                                    });
 
-            var json = JsonConvert.SerializeObject(callTargetIntegrations.Concat(callSiteIntegrations), serializerSettings);
+            var json = JsonConvert.SerializeObject(callSiteIntegrations, serializerSettings);
 
             foreach (var outputDirectory in outputDirectories)
             {
                 var filename = Path.Combine(outputDirectory, "integrations.json");
                 var utf8NoBom = new UTF8Encoding(encoderShouldEmitUTF8Identifier: false);
                 Console.WriteLine($"Writing {filename}...");
                 File.WriteAllText(filename, json, utf8NoBom);
+                
+                // CallTarget
+                var calltargetPath = Path.Combine(outputDirectory, "src", "Datadog.Trace", "ClrProfiler", "InstrumentationDefinitions.cs");
+                Console.WriteLine($"Writing {calltargetPath}...");
+                File.WriteAllText(calltargetPath, callTargetFileContent, utf8NoBom);
             }
         }
 
-        static IEnumerable<Integration> GetCallTargetIntegrations(ICollection<Assembly> assemblies)
+        static string CreateCallTargetDefinitionFileContent(IEnumerable<CallTargetDefinitionSource> callTargetIntegrations)
+        {
+            var cTargetIntegrations = callTargetIntegrations.Distinct().ToList();
+            var sBuilder = new StringBuilder();
```

Comment:
Is it worth changing this to write directly to a (file) stream instead of building the entire string in memory first? I would request the change if this was production code, but it's just a CI tool, so level of care low.

Reply:
Sure, I like the idea, will do the change.

Reply:
(don't bother changing the call site json to do this, it will go away soon)

Reply:
done in 73f5a81